### ISSUE TITLE
Allow single line position imput

### ISSUE THREAD

**Issue by jidanni** [MarcusWolschon/osmeditor4android#2058]
Here one must break the input into two lines.![Screenshot_20230125-184534_Vespucci.jpg](https://user-images.githubusercontent.com/1230959/214544326-ccd731fc-f7d6-4f1d-98f7-ae1c70512790.jpg)

It would be great if there was an additional box to enter it all in one line, like this:
![Screenshot_20230125-184656_Vespucci.jpg](https://user-images.githubusercontent.com/1230959/214544491-5177cf07-c779-4373-8a03-414472395089.jpg)

**Comment by simonpoole** [comment]
The whole point of having two separate fields is to avoid the vagaries of parsing a coordinate string, if that goes wrong in the "Go to coordinates" function it is harmless, not so changing the coordinates of a node.

**Comment by jidanni** [comment]
Yes, depending what people have in their clipboards, one method might be less hazard prone than the other.
That's why offering both is the best way:

- Latitude ________
- Longitude _______
- Single line _______

I had much more trouble pasting Y,X into Latitude, and then trying to remove the comma and the X and then paste the X into Longitude. I mean these days we get our Y,X in one string... so no need to go back and forth between apps to first get the X and then get the Y. But some folks like doing that. Anyway, in the lower right image on https://www.offline-maps.net/en/help/search we see he has a checkbox to change the field.

You might say well then both of the images I posted above should have three input fields. OK. Makes sense.
